admin/sql/create_tables.sql

```diff
@@ -154,4 +154,273 @@ CREATE TABLE feedback (
   suggestion         TEXT
 );
 
+
+CREATE SCHEMA IF NOT EXISTS musicbrainz;
```

Comment:
this should be done in `create_musicbrainz_schema.sql`.

Reply:
Done!